src/components/Card/CardHeader.tsx

```diff
@@ -57,49 +68,61 @@ const CardHeader = React.forwardRef<HTMLDivElement, CardHeaderProps>(
       actionsButtonLabel = 'Actions menu',
       title,
       subtitle,
-      adornment,
+      leftAdornment = null,
+      onHelpClick,
+      titleLinesCount,
+      subtitleLinesCount,
     },
     ref,
-  ) => {
-    return (
-      <Inline
-        ref={ref}
-        gap={SpaceSizes.md}
-        stretch={InlineEnums.StretchEnum.start}
-      >
-        <Inline align="center" gap={SpaceSizes.md}>
-          {isNotUndefined(adornment) && adornment}
-          <Stack gap={SpaceSizes.xs}>
-            <Title>{title}</Title>
-            <Text variant="secondary">{subtitle}</Text>
-          </Stack>
-        </Inline>
-
+  ) => (
+    <CardContainer
+      horizontalPadding={SpaceSizes.mdPlus}
+      verticalPadding={SpaceSizes.md}
+    >
+      <Inline ref={ref} gap={SpaceSizes.sm} stretch={leftAdornment ? 2 : 1}>
+        {leftAdornment}
+        <Stack gap={SpaceSizes.xs}>
+          <Title numberOfLines={titleLinesCount}>{title}</Title>
+          <Subtitle numberOfLines={subtitleLinesCount}>{subtitle}</Subtitle>
+        </Stack>
+        {onHelpClick && (
+          <ActionButton onClick={onHelpClick}>
+            <IconAdornmentWrapper paddingSize={SpaceSizes.sm}>
+              <Icon color="neutral.800" name="question-circle" />
+            </IconAdornmentWrapper>
+          </ActionButton>
+        )}
         {isNotUndefined(actions) && (
           <ActionsWrapper>
             <DropdownMenu actions={actions} placement="bottom-end">
               <ActionButton
                 aria-label={actionsButtonLabel}
                 title={actionsButtonLabel}
               >
-                <IconWrapper>
+                <IconAdornmentWrapper
+                  paddingSize={SpaceSizes.md}
+                  paddingType={PaddingTypes.squish}
+                >
                   <Icon name={SSCIconNames.ellipsisV} />
```

Comment:
I missed that also. But the button should be now using the horizontal ellipsis icon.